### PR TITLE
Lab 5. Added unit tests

### PR DESCRIPTION
**Pull request by andylvua**
## Coverage: 100% Methods <br> <br> Test report:
Test Results — All in Java-Labs

### CODE REVIEW

File: src/test/java/ua/lviv/iot/manager/FilmStudioTest.java [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+			true,20,50,4500, "Forza 60"),
+			1,30,"Lighting");
+	EquipmentBox equipmentBox3 = new EquipmentBox(new Decorations("China Decorations LTD.",
+			true, 45000, "Historical"),
+			1,560,"Decorations");
+
+	@BeforeEach
+	void addEquipment() {
+		studio.addNewEquipment(equipmentBox, equipmentBox1, equipmentBox2, equipmentBox3);
+	}
```
Comment (VladyslavGolovatyi) [inline src/test/java/ua/lviv/iot/manager/FilmStudioTest.java:33]: **BeforeEach** can be replaced by **BeforeAll**(make studio static before)

**Review by VladyslavGolovatyi — COMMENTED** [review]
You can add static import of some Assertions class methods to avoid duplicate code

**Review by wertylu — APPROVED** [review]

**Comment by wertylu** [comment]
все ок

**Review by VladyslavGolovatyi — APPROVED** [review]

**Comment by VladyslavGolovatyi** [comment]
Well Done

**Pull request merged**